lib/render.js

```diff
@@ -45,7 +46,7 @@ module.exports = function (html, options) {
     }
   }
 
-  var parser = MD(mdOptions).use(lazyHeaders).use(emoji, {shortcuts: {}})
+  var parser = MD(mdOptions).use(lazyHeaders).use(emoji, {shortcuts: {}}).use(expandTabs, {tabWidth: 4})
```

Comment:
2 seems to be more Node-y, but from what I'd seen on github, they go with 4 on their markdown rendering, so I chose 4 here. I'm happy to go with whatever you like. "GitHub parity" doesn't necessarily have to mean 100% identical whitespace 😀